client/src/lib/timer/hasTimerStarted.js

```diff
@@ -0,0 +1,11 @@
+export default function hasTimerStarted() {
+  const timerValue = localStorage.getItem('mosaic-timer');
+  if (
+    timerValue !== undefined
+    &&
+    timerValue !== null
+    &&
+    !isNaN(Number(timerValue))
```

Comment:
I suggest checking out the lodash functions for things like this. They have ``isFinite`` and similar which should do what you want.